manual/manual/tutorials/coreexamples.etex

```diff
@@ -357,9 +363,9 @@ let ( := ) r newval = r.contents <- newval;;
 \end{caml_example}
 
 In some special cases, you may need to store a polymorphic function in
-a data structure, keeping its polymorphism.  Without user-provided
-type annotations, this is not allowed, as polymorphism is only
-introduced on a global level.  However, you can give explicitly
+a data structure, keeping its polymorphism.  Doing this requires
+user-provided type annotation, since polymorphism is only introduced
```

Comment:
Either "Doing this requires **a** user-provided type annotation" or "Doing this requires user-provided type **annotations**" (or the third possibility "Doing this requires user-providing type **annotating**", however I'm neutral on the first two options, but dislike this one!)